vs-1.0.0-beta4/Dockerfile

```diff
@@ -1,36 +1 @@
-FROM mono:3.12
```

Comment:
there's no need for this file anymore.

I will make this point to `1.0.0-beta4` folder on Docker Hub settings once merged and 1.0.0-beta4 and vs-1.0.0-beta4 will be the same image.